修改链接&修复格式

asukaminato0721 opened the pull request:
<!-- ⚠ 请确保 Title 能清晰、准确地概括你的工作！ -->

<!-- ⚠ 如果这是一个进行中的工作 (work in progress, WIP)，请创建为 draft pull request！ -->

## 摘要
<!-- 请在这里简要说明此 PR。 -->
<!--
⚠ 建议先用列表概述，之后，如有需要，再添加几段描述。

⚠ 如果这是一个 WIP，建议使用 checklist (task list) 列举事项，例如

* [x] 这是已完成的工作。
* [ ] 这是进行中的工作。
-->

1. 因为 [issue29](https://github.com/TieBaMma/InstallTutorial/issues/29#issue-594298821) 里反馈 奶牛盘 下载收费了，故去掉了链接。

2. 修改了 OneDrive 网盘的链接，增加了“国内能访问”的说明。

3. 根据 [template](https://github.com/TieBaMma/InstallTutorial/pull/28#issuecomment-609367836) 修改了 12.1 链接的格式。

<!--## 已知问题-->
<!-- （可选）如果提交的内容中存在问题，例如很可能错误、过时的叙述，且暂时无法解决，请在这里说明。否则，请删去这一部分。 -->

<!--## 其他信息 -->
<!-- （可选）请在这里进一步说明此 PR，或者介绍相关信息。否则，请删去这一部分。 -->

Commit:
Merge pull request #4 from TieBaMma/gh-pages

跟上 main 分支

Commit:
修改了 2 个链接

修改了 OneDrive 的链接，因为奶牛盘要付费了，加了说明。

Commit:
修复了格式

按照 https://github.com/TieBaMma/InstallTutorial/pull/28#issuecomment-609367836 里的模板把 12.1 的链接全部改写了。有些地方用了注释。

Lemmingh commented on index.md:
````diff
@@ -89,21 +89,28 @@ Wolfram Engine 的激活方法可参考 [官网的介绍](http://support.wolfram
+<!--  * 磁力链接：-->
 
-[Mathematica 12.1.0 英文版 (Win 版）+ Mac 版下载器 + Win 版下载器 , 百度盘链接](https://pan.baidu.com/s/1d8mwxiEFqNI7JO_-c8smyg) 提取码：`1234`
+ <!--   ```uri-->
+ <!--   magnet:?xxxx-->
+ <!--   ```-->
````
```suggestion
```

不存在的东西，不需要留占位，这几行删掉即可。😅

其他几处：

* 行 94
* 行 109
* 行 105 末尾的括号

请自行处理。GitHub 的 diff 实在迷惑，留个 comment 很费劲。

Commit:
去除了不必要的占位

根据 https://github.com/TieBaMma/InstallTutorial/pull/32#discussion_r405226792 的建议去除了占位。

Lemmingh reviewed (approved):
✔ LGTM.

@TieBaMma , **squash merge** 吧。
